server/backend/migration/20231003000000.go

```diff
@@ -0,0 +1,193 @@
+		}
+
+		if err := db.Model(&model.DishNameTagOption{}).Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).Create(&parent).Error; err != nil {
+			fields := log.Fields{"en": v.TagNameEnglish, "de": v.TagNameGerman}
+			log.WithError(err).WithFields(fields).Error("Error while creating tag")
+		}
+		parentId = parent.DishNameTagOption
+
+		addCanBeIncluded(parentId, db, v)
+		addNotIncluded(parentId, db, v)
+	}
+}
+
+func addNotIncluded(parentId int64, db *gorm.DB, v nameTag) {
+	for _, expression := range v.NotIncluded {
+		fields := log.Fields{"expression": expression, "parentId": parentId}
+		err := db.Model(&model.DishNameTagOptionExcluded{}).
+			Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).
+			Create(&model.DishNameTagOptionExcluded{
+				Expression: expression,
+				NameTagID:  parentId}).Error
+		if err != nil {
+			log.WithError(err).WithFields(fields).Error("Unable to create new can be excluded tag")
+		}
+	}
+}
+
+func addCanBeIncluded(parentId int64, db *gorm.DB, v nameTag) {
+	for _, expression := range v.CanBeIncluded {
+		fields := log.Fields{"expression": expression, "parentId": parentId}
+
+		err := db.Model(&model.DishNameTagOptionIncluded{}).
+			Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).
+			Create(&model.DishNameTagOptionIncluded{
+				Expression: expression,
+				NameTagID:  parentId,
+			}).Error
+		if err != nil {
+			log.WithError(err).WithFields(fields).Error("Unable to create new can be excluded tag")
+		}
+	}
+}
+
+/*
+Reads the json file at the given path and checks whether the values have already been inserted into the corresponding table.
+If an entry with the same German and English name exists, the entry won't be added.
+The TagType is used to identify the corresponding model
+*/
+func setTagTable(path string, db *gorm.DB, tagType backend.ModelType) {
+	tagsDish := generateRatingTagListFromFile(path)
+	insertModel := getTagModel(tagType, db)
+	for _, v := range tagsDish.MultiLanguageTags {
+		fields := log.Fields{"de": v.TagNameGerman, "en": v.TagNameEnglish}
+		var createError error
+		if tagType == backend.CAFETERIA {
+			element := model.CafeteriaRatingTagOption{
+				DE: v.TagNameGerman,
+				EN: v.TagNameEnglish,
+			}
+			createError = insertModel.Create(&element).Error
+		} else {
+			element := model.DishRatingTagOption{
+				DE: v.TagNameGerman,
+				EN: v.TagNameEnglish,
+			}
+			createError = insertModel.Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).Create(&element).Error
+		}
+
+		if createError != nil {
+			log.WithError(createError).WithFields(fields).Error("Unable to create new can be excluded tag")
+		} else {
+			log.WithFields(fields).Info("New Entry successfully created")
+		}
+	}
+}
+
+func getTagModel(tagType backend.ModelType, db *gorm.DB) *gorm.DB {
+	if tagType == backend.DISH {
+		return db.Model(&model.DishRatingTagOption{})
+	} else {
+		return db.Model(&model.CafeteriaRatingTagOption{})
+	}
```

Comment:
nit:early returns might be advantageous for readability

```suggestion
	}
	return db.Model(&model.CafeteriaRatingTagOption{})
```